### DISCUSSION TITLE
Upgrade 4.8.2 to 5.7.2

### DISCUSSION THREAD

**Discussion by nomorehugs** [firefly-iii/firefly-iii#6014] [Firefly III] [answered]
Hello all,
my installation is finally migrated to a server where I have ssh access and can make the upgrade. But I have a lot of versions and I want to ask besides using PHP 7.4, is it anything that I need to be careful about before the upgrade?

**Comment by JC5** [comment, marked as answer]
Make sure you have a backup. It should upgrade out of the box otherwise.

**Reply by nomorehugs** [reply]
Yep, I will make a backup. I will update here after I try to upgrade.

**Reply by nomorehugs** [reply]
All good, I`m with the newest version!
